### PR TITLE
Add some logging for depth tested efb color pokes.

### PR DESCRIPTION
**Pull request by phire**

### CODE REVIEW

File: Source/Core/Core/PowerPC/MMU.cpp [hunk 1]
```diff
@@ -91,7 +95,13 @@ static void EFB_Write(u32 data, u32 addr)
 	int x = (addr & 0xfff) >> 2;
 	int y = (addr >> 12) & 0x3ff;
 
-	if (addr & 0x00400000)
+	if (addr & 0x00800000)
+	{
+		// It's possible to do a z-tested write to efb by writing a 64bit value to this address range.
+		// Not much is known, but lets at least get some loging.
```
Comment (neobrain) [inline Source/Core/Core/PowerPC/MMU.cpp]: gg

Comment (neobrain) [inline Source/Core/Core/PowerPC/MMU.cpp]: let's

Comment (neobrain) [inline Source/Core/Core/PowerPC/MMU.cpp]: EFB

**Comment by neobrain** [comment]
Do we know anything which uses this feature, yet?

**Comment by phire** [comment]
[Game Boy Interface](http://www.gc-forever.com/forums/viewtopic.php?t=2782) (homebrew gameboy player interface software) apparently uses a lot of features dolphin doesn't support, including depth tested pokes.

It's possible one or two games use it, I was considering using a panic alert instead of a error log so we would get a bug report faster if something was using it.

**Pull request merged**